src/Tests/Microsoft.NET.Publish.Tests/PublishItemsOutputGroupTests.cs

```diff
@@ -26,41 +28,25 @@ public PublishItemsOutputGroupTests(ITestOutputHelper log) : base(log)
         };
 
         [Fact]
-        public void GroupPopulatedWithRid()
+        public void WithRid()
```

Comment:
Similar to the above, replace the three `[Facts]` with `[Thoery]` based on parameters of `RunPublishItemsOutputGroupTest`